components/Avatar/Avatar.tsx

```diff
@@ -19,7 +19,7 @@ const Avatar = ({
 }): JSX.Element => {
   return (
     <AvatarBody>
-      <AvatarImage src={userAvatar} alt="" />
+      <AvatarImage src={userAvatar} alt="user avatar" />
```

Comment:
refactor (a11y): `"user avatar"` is too generic. If the avatar conveys meaning, add more context, like the username, otherwise leave the alt test to `""`.

```suggestion
      <AvatarImage src={userAvatar} alt=`profile image for user ${username}` />
```

Reply:
Also, this is only a unit test task. This should be removed into an A11Y ticket. I want to keep things simple for your first test.